src/Google/Service/Resource.php

```diff
@@ -232,6 +200,37 @@ public function call($name, $arguments, $expectedClass = null)
     return $this->client->execute($request, $expectedClass);
   }
 
+  private function logServiceError($txt, $name, $parameter = null)
+  {
+    $array = array(
+        'service' => $this->serviceName,
+        'resource' => $this->resourceName,
+        'method' => $name,
+        'parameter' => $parameter
+    );
+    if ($parameter) {
+        $array['parameter'] = $parameter;
+    }
+    $this->client->getLogger()->error(
+        $txt,
+        $array
+    );
+  }
+  
+  private function logServiceInfo($txt, $name, $parameters)
+  {
+    $array  = array(
+            'service' => $this->serviceName,
+            'resource' => $this->resourceName,
+            'method' => $name,
+            'arguments' => $parameters,
+        );
```

Comment:
This indentation is off - this should be back four spaces, and the array arguments should only be indented 2 spaces.